src/AbstractSQLRules2SQL.ts

```diff
@@ -996,6 +1010,14 @@ const typeRules: Dictionary<MatchFn> = {
 			return `TIME(${date})`;
 		}
 	},
+	ToJSON: (args, indent) => {
+		checkMinArgs('ToJSON', args, 1);
+		if (engine !== Engines.postgres) {
+			throw new SyntaxError('ToJSON not supported on: ' + engine);
+		}
+		const column = TextValue(getAbstractSqlQuery(args, 0), indent);
+		return `to_json(${column})::text`;
```

Comment:
This should just convert to json
```suggestion
		return `to_json(${column})`;
```
and if necessary the caller can do
```js
['Cast', ['ToJSON', x], 'Text']
```
to cast it back to text